python/grass/temporal/univar_statistics.py

```diff
@@ -65,11 +65,10 @@ def compute_univar_stats(registered_map_info, stats_module, fs, rast_region=Fals
 
     if not univar_stats:
         gs.warning(
-            _(
-                "Unable to get statistics for {voxel}raster map "
-                "<{rmap}>".format(
-                    rmap=id, voxel="" if stats_module.name == "r.univar" else "3d "
-                )
+            _("Unable to get statistics for raster map <{rmap}>").format(rmap=id)
+            if stats_module.name == "r.univar"
+            else _("Unable to get statistics for 3d raster map <{rmap}>").format(
+                rmap=id
             )
         )
```

Comment:
The source string doesn't appear in any .pot file in the repo, and can't be found in any of the 4 versions of grasslibs.pot (since april 2022) https://github.com/OSGeo/grass/commits/main/locale/templates/grasslibs.pot

However, when searching for "Unable to get statistics for" in the repo, we see that there are 3 strings that were already translated in grasslibs_ar.po, grasslibs_es.po, grasslibs_fr.po, grasslibs_pt_BR.po, grasslibs_ru.po, and grasslibs_sl.po (6 languages). Some are marked fuzzy, some don't.

If we take for example the french file, the strings already don't match the original strings here.

https://github.com/OSGeo/grass/blob/972bd32a23156f3255025a1ed6d42c9e563960b8/locale/po/grasslibs_fr.po#L9645-L9663

```
#, python-format
#~ msgid "Unable to get statistics for raster map <%s>"
#~ msgstr "Impossible d'obtenir les statistiques de la carte raster <%s>"

#, python-format
#~ msgid "Unable to get statistics for 3d raster map <%s>"
#~ msgstr "Impossible d'obtenir les statistiques pour la carte raster 3d <%s>"

#, python-format
#~ msgid "Space time %(sp)s dataset <%(i)s> not found"
#~ msgstr "Le jeu de données <%(i)s> temporel %(sp)s n'a pas été trouvé"

#, python-format
#~ msgid "Space time %(sp)s dataset <%(i)s> is empty"
#~ msgstr "Le jeu de données <%(i)s> temporel %(sp)s est vide"

#, python-format
#~ msgid "Unable to get statistics for vector map <%s>"
#~ msgstr "Impossible d'obtenir les statistiques pour la carte vecteur <%s>"
```



So, probably to make the existing translation match up again, it should be changed back to

```suggestion
    if not univar_stats:
        gs.warning(
            _("Unable to get statistics for raster map <%s>") % id
            if stats_module.name == "r.univar"
            else _("Unable to get statistics for 3d raster map <%s>") % id
        )
```


However, what I suggested for my PR makes it equivalent to the what was before this PR, if the intented formatting of `{voxel}` was applied before getting the translated string, but `{rmap}` was kept as a placeholder. But even in that case, the string doesn't exist in the .pot file.

Reply:
The original code is handling the strings in a way which assumes you can construct a sentence using string operations. This may work for some languages but not others and it seems to me as impossible to decipher when translating the strings by themselves. Having two complete sentences is the best practice here.

Reply:
I applied the change that will allow to match up existing translations to that code